addon/components/fast-link-to.js

```diff
@@ -1,286 +1,49 @@
 import Ember from 'ember';
 import layout from '../templates/components/fast-link-to';
 
-let { assert, get, inject, deprecate, ControllerMixin } = Ember;
+let { get } = Ember;
 let EmberComponent = Ember.Component;
 
+var linkProto = Ember.LinkComponent.proto();
+var properties = Object.keys(linkProto);
+var fastLinkProto = {};
 
-let FastLinkComponent = EmberComponent.extend({
-  layout: layout,
+for (var i = 0; i < properties.length; i++) {
```

Comment:
Because I didn't think of it. That being said:
- `Ember.assign` wasn't a thing in 1.13.
- `Object.assign` isn't in everything Ember supports.

Could do `let collapse = Ember.assign || Ember.merge;`

/shrug